src/test/java/core/basesyntax/service/fileservice/FileWriterServiceImplTest.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.service.fileservice;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileWriterServiceImplTest {
+    private static String pathToTestReportFile;
+    private static String invalidFilePath;
+    private static String report;
+    private static FileWriterService writerService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        pathToTestReportFile = "src/main/resources/testReport.csv";
+        invalidFilePath = "";
+        report = "fruit,quantity\n"
+                + "banana,352\n"
```

Comment:
i think we should use System.lineSeparator() insted of "\n" in case that our test might be executed on macOS for example